src/Neo/SmartContract/Native/NativeContract.cs

```diff
@@ -288,13 +283,26 @@ internal bool IsInitializeBlock(ProtocolSettings settings, uint index, out Hardf
 
                 if (activeIn == index)
                 {
-                    hardfork = hf;
-                    return true;
+                    hfs.Add(hf);
                 }
             }
 
+            // Return all initialize hardforks
+            if (hfs.Count > 0)
+            {
+                hardforks = hfs.ToArray();
+                return true;
+            }
+
+            // If is not configured, the Genesis is an initialization block.
+            if (index == 0 && ActiveIn is null)
+            {
+                hardforks = hfs.ToArray();
```

Comment:
```suggestion
                hardforks = null; // no hardforks are active at this point (according to the check above).
```

Or, if you're against of this change, then we need to adjust line 81 of `TestIsInitializeBlock`.

Reply:
I prefer return all of them, what do you think @neo-project/core ?

Reply:
> I prefer return all of them

It’s perfectly OK, then we just need to adjust the test instead of the suggested change.

Reply:
Test fixed.